--- a/Tools/installHaxx/Package.swift	2022-11-08 15:51:25
+++ b/Tools/installHaxx/Package.swift	2022-11-08 15:59:46
@@ -17,8 +17,8 @@
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(name: "SwiftUtils", url: "https://github.com/pinauten/SwiftUtils", .branch("master")),
-        .package(name: "SwiftMachO", url: "https://github.com/pinauten/SwiftMachO", .branch("master"))
+        .package(name: "SwiftUtils", path: "../../OfflinePackages/SwiftUtils"),
+        .package(name: "SwiftMachO", path: "../../OfflinePackages/SwiftMachO")
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
--- a/Tools/Exe2Driver/Package.swift	2022-11-08 15:51:25
+++ b/Tools/Exe2Driver/Package.swift	2022-11-08 15:59:32
@@ -16,8 +16,8 @@
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(name: "SwiftUtils", url: "https://github.com/pinauten/SwiftUtils", .branch("master")),
-        .package(name: "SwiftMachO", url: "https://github.com/pinauten/SwiftMachO", .branch("master"))
+        .package(name: "SwiftUtils", path: "../../OfflinePackages/SwiftUtils"),
+        .package(name: "SwiftMachO", path: "../../OfflinePackages/SwiftMachO")
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
--- a/FuguInstall/Package.swift	2022-11-08 15:51:25
+++ b/FuguInstall/Package.swift	2022-11-08 15:58:13
@@ -17,7 +17,7 @@
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(url: "https://github.com/weichsel/ZIPFoundation.git", .upToNextMajor(from: "0.9.0"))
+        .package(name: "ZIPFoundation", path: "../OfflinePackages/ZipFoundation")
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
--- a/Packages/Fugu15KernelExploit/Package.swift	2022-11-08 15:51:25
+++ b/Packages/Fugu15KernelExploit/Package.swift	2022-11-08 15:59:06
@@ -16,11 +16,11 @@
             targets: ["Fugu15KernelExploit"])
     ],
     dependencies: [
-        .package(url: "https://github.com/pinauten/SwiftUtils", .branch("master")),
+        .package(path: "../../OfflinePackages/SwiftUtils"),
         .package(path: "../ProcessCommunication"),
-        .package(url: "https://github.com/pinauten/PatchfinderUtils", .branch("master")),
-        .package(url: "https://github.com/pinauten/KernelPatchfinder", .branch("master")),
-        .package(url: "https://github.com/pinauten/iDownload", .branch("master")),
+        .package(path: "../../OfflinePackages/PatchfinderUtils"),
+        .package(path: "../../OfflinePackages/KernelPatchfinder"),
+        .package(path: "../../OfflinePackages/iDownload"),
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
--- a/OfflinePackages/iDownload/Package.swift	2022-11-08 15:54:57
+++ b/OfflinePackages/iDownload/Package.swift	2022-11-08 16:03:17
@@ -17,7 +17,7 @@
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(name: "SwiftUtils", url: "https://github.com/pinauten/SwiftUtils", .branch("master"))
+        .package(name: "SwiftUtils", path: "../SwiftUtils")
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
--- a/OfflinePackages/SwiftMachO/Package.swift	2022-11-08 15:54:55
+++ b/OfflinePackages/SwiftMachO/Package.swift	2022-11-08 16:03:53
@@ -12,7 +12,7 @@
             targets: ["SwiftMachO"]),
     ],
     dependencies: [
-        .package(name: "SwiftUtils", url: "https://github.com/pinauten/SwiftUtils", .branch("master"))
+        .package(name: "SwiftUtils", path: "../SwiftUtils")
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
--- a/OfflinePackages/KernelPatchfinder/Package.swift	2022-11-08 15:54:57
+++ b/OfflinePackages/KernelPatchfinder/Package.swift	2022-11-08 16:03:04
@@ -18,9 +18,9 @@
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(name: "SwiftUtils", url: "https://github.com/pinauten/SwiftUtils", .branch("master")),
-        .package(name: "SwiftMachO", url: "https://github.com/pinauten/SwiftMachO", .branch("master")),
-        .package(name: "PatchfinderUtils", url: "https://github.com/pinauten/PatchfinderUtils", .branch("master"))
+        .package(name: "SwiftUtils", path: "../SwiftUtils"),
+        .package(name: "SwiftMachO", path: "../SwiftMachO"),
+        .package(name: "PatchfinderUtils", path: "../PatchfinderUtils")
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
--- a/OfflinePackages/PatchfinderUtils/Package.swift	2022-11-08 15:54:56
+++ b/OfflinePackages/PatchfinderUtils/Package.swift	2022-11-08 16:03:42
@@ -17,8 +17,8 @@
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(name: "SwiftUtils", url: "https://github.com/pinauten/SwiftUtils", .branch("master")),
-        .package(name: "SwiftMachO", url: "https://github.com/pinauten/SwiftMachO", .branch("master"))
+        .package(name: "SwiftUtils", path: "../SwiftUtils"),
+        .package(name: "SwiftMachO", path: "../SwiftMachO")
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
